static/app/views/settings/organizationIntegrations/configureIntegration.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import ConfigureIntegration from 'sentry/views/settings/organizationIntegrations/configureIntegration';
+
+describe('OpsgenieMigrationButton', function () {
+  const org = TestStubs.Organization({
+    access: ['org:integrations', 'org:write'],
+  });
+  const integrationId = '1';
+  it('Migrate Plugin button hits migration endpoint', async function () {
+    org.features.push('integrations-opsgenie-migration');
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/config/integrations/`,
+      body: {
```

Comment:
it would probably be good to pull this out and put it into its own `TestStub` like `JiraIntegrationProvider` https://github.com/getsentry/sentry/blob/master/fixtures/js-stubs/jiraIntegrationProvider.tsx

you could also delete anything that isn't strictly necessary to perform the plugin migration